Benqi-Agents/Large-Delegate-Votes-Change/README.md

```diff
@@ -0,0 +1,37 @@
+# Delegate Votes Monitor Agent
+
+## Description
+
+This agent detects large (>= 30%) increments in a delegate account's QI token
+vote balance by monitoring `DelegateVotesChanged` events emitted by the token
+contract.
+
+## Supported Chains
+
+- Avalanche
+
+## Alerts
+
+- BENQI-1-1
+  - Fired when a delegate account's vote balance increases by 30% or more.
+  - Severity is always set to "Info"
```

Comment:
The agent takes the percentage as a parameter, so avoid using `30%` in describing the agent and the finding, referring to this as a large increase should be enough.